### PR TITLE
ci: update the list of excluded paths in NgBot config

### PR DESCRIPTION
**Pull request by AndrewKushnir**
This commit aligns the NgBot config that is used to determine whether a presubmit is needed with [its internal version](google3/third_party/javascript/angular2/copy.bara.sky).

## PR Type
What kind of change does this PR introduce?

- [x] CI related changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

File: .github/angular-robot.yml [hunk 1]
```diff
@@ -42,9 +42,13 @@ merge:
     exclude:
       - 'packages/*'
       - 'packages/bazel/*'
+      - 'packages/bazel/src/*'
       - 'packages/bazel/src/api-extractor/**'
+      - 'packages/bazel/src/builders/**'
       - 'packages/bazel/src/ng_package/**'
-      - 'packages/bazel/src/ng_module/**'
```
Comment (AndrewKushnir) [inline .github/angular-robot.yml:47]: Note to the reviewers: this is a bit unexpected: we were syncing the code from this path, but the presubmit was not required (the path was excluded)...

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 3e7d2991570a7fd8879a87ffafa77936c41a30dd.

File: .github/angular-robot.yml (hunk 1 above)
Comment (devversion) [reply .github/angular-robot.yml:47]: Ahh interesting, I think we chatted about this and this rule is no longer used internally, so potentially we can stop syncing

**Pull request closed without merging**